test/compflags/sungeun/configs/type_variables/configTypeFun.chpl

```diff
@@ -12,11 +12,16 @@ class cPair {
 
 config type myType = rPair;
 
-proc f(p:myType?) {
+proc toNonNilableIfClassType(type arg) type {
+  if isClassType(arg) then return arg?;
```

Comment:
Shouldn't these helpers be named `toNilableIf...`?

Reply:
Oops. I will adjust the PR text.